### DISCUSSION TITLE
511797CB-E938-DB68-E787-C2749E6A1ECC

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#92] [General]
# M 42 (Orion Nebula)

The Orion Nebula, also known as Messier 42 or NGC 1976, is a diffuse nebula about 1,344 light-years away in the constellation of Orion. It is one of the brightest nebulae and is visible to the naked eye in the night sky. This nebula is the closest region of massive star formation to Earth

https://celestia.mobi/addon?item=511797CB-E938-DB68-E787-C2749E6A1ECC&lang=en

**Comment by exocritic5457** [comment]
Any Carina addons?

**Reply by kmjordan122313-lgtm** [reply]
this is good ^ _ ^
